instrumentation/openai-client-instrumentation/instrumentation-0.8/src/main/java/co/elastic/otel/openai/v0_2/ApiAdapterImpl.java

```diff
@@ -61,11 +65,11 @@ public String extractText(ChatCompletionContentPart part) {
   @Override
   public String extractType(ChatCompletionCreateParams.ResponseFormat val) {
     if (val.isResponseFormatText()) {
-      return val.asResponseFormatText()._type().toString();
+      return "text";
     } else if (val.isResponseFormatJsonObject()) {
-      return val.asResponseFormatJsonObject()._type().toString();
+      return "json_object";
     } else if (val.isResponseFormatJsonSchema()) {
-      return val.asResponseFormatJsonSchema()._type().toString();
+      return "json_schema";
```

Comment:
For some versions `_type()` returns null here, so I instead hardcoded the strings for these older versions.